src/react-native/18-maps/maps.md

```diff
@@ -2,25 +2,67 @@
 @parent learn-react-native 18
 @outline 3
 
-@description TODO
+@description Learn how to integrate Google Maps into your React Native application.
 
 @body
 
 ## Overview
 
 In this section, you will:
 
-- TODO
+- Integrate Google Maps
+- Be introduced to gradle
 
 ## Objective 1: Add Google Maps to Restaurant List page
 
-<img alt="Screenshot of the restaurant view with the title “Green Bay, Wisconsin.” There are two tabs at the top, List and Map, with Map selected. The map below is centered on Green Bay with nearby locations labeled. The bottom tab bar has icons for Place My Order and Settings." src="../static/img/react-native/18-maps/02-solution.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
+<img alt="Screenshot of the restaurant view with the title “Green Bay, Wisconsin.” There are two tabs at the top, List and Map, with Map selected. The map below is centered on Green Bay. The bottom tab bar has icons for Place My Order and Settings." src="../static/img/react-native/18-maps/01-solution.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
 
-TODO
+### React Native Maps
 
-### Concept TODO
+In this section, we will be using the `react-native-maps` library to integrate Google Maps into our application. This library provides several React Native components that can be used to render maps on both iOS and Android.
```

Comment:
I would mention that it includes a lot of components for building maps (polygons, custom map styles, etc.) and we will focus on just the map and marker components.